web/src/main/scala/net/wiringbits/Main.scala

```diff
@@ -8,12 +8,12 @@ import net.wiringbits.webapp.utils.slinkyUtils.components.core.{ErrorBoundaryCom
 import org.scalajs.dom
 import slinky.hot
 import slinky.web.ReactDOM
-
+import concurrent.ExecutionContext.Implicits.global
 import scala.scalajs.js.annotation.JSImport
 import scala.scalajs.{LinkingInfo, js}
-
-@JSImport("js/index.css", JSImport.Default)
```

Comment:
Error: Cannot find module 'js/index.css'
Require stack:
- /Users/aksuristola/Desktop/scala-webapp-template/web/target/scala-3.3.0-RC3/scalajs-bundler/main/wiringbits-web-fastopt.js
- /Users/aksuristola/Desktop/scala-webapp-template/[stdin]
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/Users/aksuristola/Desktop/scala-webapp-template/web/target/scala-3.3.0-RC3/scalajs-bundler/main/wiringbits-web-fastopt.js:29:31)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/aksuristola/Desktop/scala-webapp-template/web/target/scala-3.3.0-RC3/scalajs-bundler/main/wiringbits-web-fastopt.js',
    '/Users/aksuristola/Desktop/scala-webapp-template/[stdin]'
  ]
}
[error] org.scalajs.jsenv.ExternalJSRun$NonZeroExitException: exited with code 1
[error]         at org.scalajs.jsenv.ExternalJSRun$$anon$1.run(ExternalJSRun.scala:200)
[error] stack trace is suppressed; run last web / Compile / run for the full output
[error] (web / Compile / run) org.scalajs.jsenv.ExternalJSRun$NonZeroExitException: exited with code 1
[error] Total time: 7 s, completed 19 Jun 2023, 19.43.01